standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/DirectSqlUpdateStat.java

```diff
@@ -229,12 +229,11 @@ private void insertIntoPartColStatTable(Map<PartColNameInfo, MPartitionColumnSta
                                           Connection dbConn) throws SQLException, MetaException, NoSuchObjectException {
     PreparedStatement preparedStatement = null;
     int numRows = 0;
-    String insert = "INSERT INTO \"PART_COL_STATS\" (\"CS_ID\", \"CAT_NAME\", \"DB_NAME\","
-            + "\"TABLE_NAME\", \"PARTITION_NAME\", \"COLUMN_NAME\", \"COLUMN_TYPE\", \"PART_ID\","
+    String insert = "INSERT INTO \"PART_COL_STATS\" (\"CS_ID\", \"COLUMN_NAME\", \"COLUMN_TYPE\", \"PART_ID\","
```

Comment:
you have another PR that renamed and refactored this class, right?

Reply:
Yes, it was renamed to `DirectSqlUpdatePart.java`.